test/integration/Makefile

```diff
@@ -0,0 +1,30 @@
+SIS_NAME = sis
+SIS_VERSION = 2.29
+
+SRC_DIR = src
+BUILD_DIR = ../../build
+RTEMS_APP_DIR = /opt/rtems-6-sparc-gr712rc-smp-4/src/example/b-gr712rc-qual-only/app.exe
+RESOURCES_DIR = resources/expectedResult
+
+EXPECTED := "$(shell cat $(RESOURCES_DIR))"
+RESULT := "$(shell ./$(BUILD_DIR)/$(SRC_DIR)/$(SIS_NAME)-$(SIS_VERSION) -dumbio -r $(RTEMS_APP_DIR))"
+TESTS = run_rtems
+
+check: ${TESTS}
+
+.SILENT :
+	${TESTS}
+
+.PHONY :
+	${TESTS}
+	check
+
+##############################INTEGRATION TESTS################################
+
+run_rtems:
```

Comment:
Please correct me if I am wrong, but the actual test is executed when the RESULT variable is assigned, right? I would prefer it to be executed within the rule

Also, if we are to include more integration tests in the future, either the file should be prepared to handle more, or we should prepare to have more makefiles.

In other tests done in makefile, we simply propagate the failure of a grep. Or, we can do exit 1.

Reply:
Ack